kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1348,6 +1348,16 @@ object KyuubiConf {
     .version("1.2.0")
     .fallbackConf(LEGACY_ENGINE_SHARE_LEVEL)
 
+  val ENGINE_SHARE_LEVEL_GROUP_NAME: ConfigEntry[Option[String]] =
+    buildConf("kyuubi.engine.share.level.group.name")
```

Comment:
How about "kyuubi.engine.share.level.group.primaries"(`ConfigEntry[Seq[String]]`)?
And we can define the prefered list of group names.

Reply:
cc @yaooqinn